pulser-core/pulser/json/abstract_repr/deserializer.py

```diff
@@ -381,6 +382,28 @@ def _deserialize_register(
     return reg
 
 
+def _deserialize_noise_model(noise_model_obj: dict[str, Any]) -> NoiseModel:
+
+    def convert_complex(obj: list | tuple) -> list:
+        if isinstance(obj, (list, tuple)):
+            return [convert_complex(e) for e in obj]
+        elif isinstance(obj, dict):
+            return obj["real"] + 1j * obj["imag"]
+        else:
+            return obj
+
+    eff_noise_rates = []
+    eff_noise_opers = []
+    for rate, oper in noise_model_obj.pop("eff_noise"):
+        eff_noise_rates.append(rate)
+        eff_noise_opers.append(convert_complex(oper))
+    return pulser.NoiseModel(
+        **noise_model_obj,
+        eff_noise_rates=tuple(eff_noise_rates),
+        eff_noise_opers=tuple(eff_noise_opers),
+    )
```

Comment:
I think this will fail if noise_model_obj has eff_noise_rates or eff_noise_opers defined, 
```suggestion
    noise_model_obj.update(**dict(
        eff_noise_rates=tuple(eff_noise_rates),
        eff_noise_opers=tuple(eff_noise_opers),
        )
    )
    return pulser.NoiseModel(
        **noise_model_obj
    )
```

Reply:
I guess we can never end up in this case since we perform the validation before using this function...

Reply:
Exactly, the schema is taking care of this